src/Core/Utils/Attribs.hpp

```diff
@@ -27,7 +27,7 @@ class AttribBase {
     void setName( std::string name ) { m_name = name; }
     virtual void resize( size_t s ) = 0;
 
-    virtual uint getSize() = 0;
+    virtual size_t getSize() = 0;
```

Comment:
add documentation

Reply:
well this one is to be used along with stride to send data to the gpu. 
Actually it is not used yet ... (size is getted from VecArray::Vector::RowsAtCompileTime with the sendGL method of Engine::Mesh).

Nevertheless we could imagine having both meth (get size : number of attributes and get Whatever : dim of each value).

But this function is clearly related to gpu stuff, and size should always be the number of vertices in the mesh (so user don't have to worry about one attrib specific size).